user-export.sh

```diff
@@ -25,7 +44,7 @@ do
   mkdir -p $WORK_DIRECTORY/$username
 
   # Get full user json excluding the id and timestamp
-  kcadm.sh get users/$user_id --fields '*(*(*(*(*(*))))),-id,-createdTimestamp' > $WORK_DIRECTORY/$username/user.json
+  kcadm.sh get users/$user_id --fields '*(*(*(*(*(*))))),-id,-createdTimestamp' --realm $EXPORT_REALM --server $EXPORT_KEYCLOAK_SERVER --token $EXPORT_TOKEN --no-config > $WORK_DIRECTORY/$username/USER.json
 
   # If failed to get user, skip the current iteration
   if [ $? -ne 0 ]; then
```

Comment:
Consider writing to a log file

Reply:
sure, will make it through the Makefile 
(container output redirect), where ir un the container in interactive mode